src/components/atoms/Avatar/Avatar.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './Avatar.scss'
+
+const Avatar = ({ image, alt, size, className, style, ...props }) => (
+  <img
+    src={image}
+    alt={alt}
```

Comment:
We can keep the src and alt prop as it is since it's a img tag attributes we can keep as it is and just merge the props
like:

```
<img
 className={`cg-avatar ${className}`}
    style={{ width: size, height: size, ...style }}
    {...props}
  ></img>
```
props will have the src and alt